IaC/AKS/README.md

```diff
@@ -330,6 +332,49 @@ kubectl create secret generic ngsa-aks-secrets \
 
 ```
 
+## Setup SSL/DNS
+
+### DNS Setup
+
+Assuming that you have a domain name, and permissions to update DNS records for your domain, create an A record to point your domain to `INGRESS_PIP`.
+
+For example, in Azure:
+
+```bash
+
+az network dns record-set a add-record -g <dns zone resource group> -z <dns zone name> -n $Ngsa_Name -a $INGRESS_PIP --ttl <TTL seconds>
+
+```
+
+Set the application endpoint
+
+```bash
+
+export Ngsa_App_Endpoint=replaceWithDNSHostname
```

Comment:
change `replacewithDNSHostname` to <DNS_Host_Name> or something similar to make it visible that this value needs to be changed